src/containers/teamPage/index.tsx

```diff
@@ -0,0 +1,268 @@
+  display: inline-block;
+  color: ${BLACK};
+  font-size: 15px;
+  line-height: 0.5;
+`;
+
+const StyledPanel = styled(Panel)`
+  min-height: 40px;
+  margin-bottom: 10px;
+  padding-left: 10px;
+  background-color: ${LIGHT_GREEN}90;
+  border: 0px;
+`;
+
+const Line = styled.div`
+  height: 1px;
+  width: 104%;
+  margin: 0px -10px 10px -20px;
+  display: block;
+  background-color: ${WHITE};
+`;
+
+// dummy data
+const sampleMemberData: MemberProps[] = [
+  {
+    user_id: 1,
+    username: 'florisdobber',
+    team_role: TeamRole.LEADER,
+  },
+  {
+    user_id: 2,
+    username: 'jackblanc',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 3,
+    username: 'atreecounter',
+    team_role: TeamRole.NONE,
+  },
+  {
+    user_id: 4,
+    username: 'speakerofthetrees',
+    team_role: TeamRole.PENDING,
+  },
+  {
+    user_id: 5,
+    username: 'vrushalitarte01',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 6,
+    username: 'willmt80',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 7,
+    username: 'apradhan12',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 8,
+    username: 'SofieDunt',
+    team_role: TeamRole.MEMBER,
+  },
+];
+
+const sampleGoalData: GoalProps[] = [
+  {
+    id: 1,
+    goal: 50,
+    progress: 5,
+    start_date: new Date(2021, 0, 1),
+    complete_by: new Date(2022, 0, 1),
+    completion_date: null,
+  },
+  {
+    id: 2,
+    goal: 500,
+    progress: 71,
+    start_date: new Date(2021, 2, 20),
+    complete_by: new Date(2021, 4, 21),
+    completion_date: null,
+  },
+  {
+    id: 3,
+    goal: 10,
+    progress: 10,
+    start_date: new Date(2020, 5, 10),
+    complete_by: new Date(2020, 7, 10),
+    completion_date: new Date(2020, 6, 13),
+  },
+];
+
+const sampleTeamData: TeamProps = {
+  id: 1,
+  name: 'Code4Community',
+  bio: 'Short description about how cool of a team Code4Community is!',
+  members: sampleMemberData,
+  goals: sampleGoalData,
+};
+
+const TeamPage: React.FC = () => {
+  const onClick = () => {
+    // TODO: join team
+  };
+
+  return (
+    <TeamContainer>
+      <StyledLinkButton to={Routes.LOGIN}>
+        {`<`} Return to Teams
+      </StyledLinkButton>
+      <TeamHeaderContainer>
+        <PageHeaderContainer>
+          <PageHeader
+            pageTitle={sampleTeamData.name}
+            pageSubtitle={sampleTeamData.bio}
+          />
+        </PageHeaderContainer>
+        <JoinButton type="primary" onClick={onClick}>
+          Join Team
+        </JoinButton>
+      </TeamHeaderContainer>
+
+      <CenterDiv>
+        <MemberContainer>
+          <SectionHeader>TEAM MEMBERS</SectionHeader>
+          <StyledMemberList
+            dataSource={sampleTeamData.members}
+            itemLayout="vertical"
+          >
+            {sampleTeamData.members.map((member) => {
+              if (
+                member.team_role === TeamRole.LEADER ||
+                member.team_role === TeamRole.MEMBER
+              ) {
```

Comment:
❓ Why is this condition required?

Reply:
The `TeamRole` enum also has types `None` (applied and was rejected or left the team) and `Pending` (has applied and not been accepted/rejected) - this is so we don't show those users